test_runner/regress/test_pageserver_catchup.py

```diff
@@ -9,7 +13,9 @@ def test_pageserver_catchup_while_compute_down(neon_env_builder: NeonEnvBuilder)
     neon_env_builder.num_safekeepers = 3
     env = neon_env_builder.init_start()
 
-    env.neon_cli.create_branch("test_pageserver_catchup_while_compute_down")
+    tenant_id = env.initial_tenant
+
+    timeline_id = env.neon_cli.create_branch("test_pageserver_catchup_while_compute_down")
```

Comment:
```suggestion
    timeline_id = env.initial_timeline
```

Could just use initial_timeline here, even though pageserver is not restarted, but in some cases having the default branch sit there without ever having a compute could be surprising in tests. I've recently been removing these branchings from tests which seem to be doing them because of an old pattern.

It is also quite slow to shell out to neon_cli.